tests/helix/send-to-helix-inner.proj

```diff
@@ -83,6 +83,8 @@
 
   <ItemGroup>
     <HelixCorrelationPayload Include="$(RepoRoot)tests/Aspire.Hosting.Azure.Tests/Snapshots" Destination="Snapshots" />
+    <HelixCorrelationPayload Include="$(RepoRoot)tests/Aspire.Hosting.Docker.Tests/Snapshots" Destination="Snapshots" />
+    <HelixCorrelationPayload Include="$(RepoRoot)tests/Aspire.Hosting.Kubernetes.Tests/Snapshots" Destination="Snapshots" />
   </ItemGroup>
```

Comment:
Ideally, this should be set in each project that uses Verify, but I couldn't work out how to do it. Perhaps @radical may have some suggestions.